cmd/kpod/history.go

```diff
@@ -0,0 +1,327 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"text/template"
+	"time"
+
+	"os"
+
+	"github.com/Sirupsen/logrus"
+	is "github.com/containers/image/storage"
+	"github.com/containers/storage"
+	units "github.com/docker/go-units"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+// historyOutputParams stores info about each layer
+type historyOutputParams struct {
+	ID        string `json:"id"`
+	Created   string `json:"created"`
```

Comment:
Same for time, I am using the HumanTime function from docker/go-units, which outputs the time elapsed as a string.